.github/workflows/build-image-ui-pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Test UI container image build and deployment
+on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
```

Comment:
agh, one last change, I'm really sorry @alexcreasy I didn't ping you there, we had a conversation in which we were blocking other branches if we did messed up the UI, so we've agreed on just running ui builds when changing something inside `clients/ui`, can you change it like this?: https://github.com/kubeflow/model-registry/pull/833/files#diff-5320b64457303894487247143c3a15570f6dca28abcb74fc4989e511c5b6aacbR6

Reply:
No worries, I've updated to use the agreed upon paths!